lib/slime/lexer.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Slime.Lexer do
+  @moduledoc """
+  Leex based lexing of Slime documents. See src/slime_lexer.xrl for details.
+  """
+
+  def tokenize(document) when is_binary document do
```

Comment:
Heh, looking further, this is too spikey to mind these details. Sorry – carry on :)